systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTopicTemplates.java

```diff
@@ -8,31 +8,27 @@
 import io.strimzi.api.kafka.model.KafkaTopicBuilder;
 import io.strimzi.operator.common.model.Labels;
 import io.strimzi.systemtest.Constants;
-import io.strimzi.systemtest.resources.ResourceManager;
+import io.strimzi.systemtest.storage.TestStorage;
 import io.strimzi.test.TestUtils;
 
 public class KafkaTopicTemplates {
 
     private KafkaTopicTemplates() {}
 
-    public static KafkaTopicBuilder topic(String clusterName, String topicName) {
-        return defaultTopic(clusterName, topicName, 1, 1, 1, ResourceManager.kubeClient().getNamespace());
+    public static KafkaTopicBuilder topic(TestStorage testStorage) {
+        return defaultTopic(testStorage.getClusterName(), testStorage.getTopicName(), 1, 1, 1, testStorage.getNamespaceName());
     }
 
     public static KafkaTopicBuilder topic(String clusterName, String topicName, String topicNamespace) {
         return defaultTopic(clusterName, topicName, 1, 1, 1, topicNamespace);
     }
 
-    public static KafkaTopicBuilder topic(String clusterName, String topicName, int partitions) {
-        return defaultTopic(clusterName, topicName, partitions, 1, 1, ResourceManager.kubeClient().getNamespace());
+    public static KafkaTopicBuilder topic(String clusterName, String topicName, int partitions, String topicNamespace) {
+        return defaultTopic(clusterName, topicName, partitions, 1, 1, topicNamespace);
     }
 
-    public static KafkaTopicBuilder topic(String clusterName, String topicName, int partitions, int replicas) {
-        return defaultTopic(clusterName, topicName, partitions, replicas, replicas, ResourceManager.kubeClient().getNamespace());
-    }
-
-    public static KafkaTopicBuilder topic(String clusterName, String topicName, int partitions, int replicas, int minIsr) {
-        return defaultTopic(clusterName, topicName, partitions, replicas, minIsr, ResourceManager.kubeClient().getNamespace());
+    public static KafkaTopicBuilder topic(String clusterName, String topicName, int partitions, int replicas, String topicNamespace) {
+        return defaultTopic(clusterName, topicName, partitions, replicas, replicas, topicNamespace);
```

Comment:
Uniting the clusterName, Name, and namespaceName parameters positions would make sense across all resources. Not strictly as part of this PR.

Reply:
Will surely do in another PR. Thanks for the tip!